### ISSUE TITLE
Unable to disable EiraIRC's nick color change function

### ISSUE THREAD

**Issue by Gunner76th** [blay09/EiraIRC#242]
I currently have 1.7.10.2.8.161 of EiraIRC on my server. I use McMyAdmin for permissions assignment and access control, and as part of that it also does username color assignments based on access permissions. Even though I have   
# If set to true, colors from this IRC context will be translated into Minecraft colors and vice-versa.
    B:convertColors=false

in the share.cfg, and no entry for this in servers.json, everyone shows up as purple in game. Are there more places that the colors are assigned, and if so how do I completely disable the nick color portion of EiraIRC?

**Comment by blay09** [comment]
To disable the name colors of Minecraft players, you can set "enableNameColors" to false.

**Comment by Gunner76th** [comment]
I am not seeing this in the shared.cfg file. is this something that is suppose to be in the servers.json file, and if so, what would the full syntax method be for the entry?

**Comment by Gunner76th** [comment]
the only entry close to the one you mentioned is 
# If set to true, players can use the '/irc color' command to set a color for their MC nick. See also: colorBlacklist
    B:enablePlayerColors=true

**Comment by blay09** [comment]
Yes, that's the one.

**Comment by Gunner76th** [comment]
I changed that line to false, and both relayed text from IRC as well as player names ingame when they speak are showing up as purple. I changed both the client and the server config lines to disabled. Here are copies of both my client and server shared.cfg files, hopefully you can find the reason my chat is still having its color modified, as well as prefixes stripped. 

Client config file: http://pastebin.com/sgThVXZ0

Server config file: http://pastebin.com/FsiG7X41

Version of EiraIRC mod: eirairc-mc1.7.10-2.8.161

Version of forge: 1291

**Comment by Gunner76th** [comment]
Also, to clarify, If the player has OP status, they are purple, anything relayed from IRC whether the user has @ + or no status comes in as purple, and if the player does NOT have OP status in game their text is white. At present I have McMyAdmin set to display admins as red, mods as yellow, normal users as green, with Admins having the Admin prefix, and Moderators having the Mod prefix. 

I tested running the server without the mod installed on either server or client, and it would seem that while EiraIRC is doing the color change to purple still, it however is not stripping the prefixes as I had thought. It would seem that for some reason McMyAdmin is failing to color code and prefix chat messages, so that is a separate issue I will have to address with them.

**Comment by blay09** [comment]
I'll check up on this tomorrow and see if there's an issue related to name colors in EiraIRC.

Prefixes being stripped might be a compatibility issue with Forge (I assume McMyAdmin is not a Forge mod?)

**Comment by blay09** [comment]
I'll check up on this tomorrow and see if there's an issue related to name colors in EiraIRC.

Prefixes being stripped might be a compatibility issue with Forge (I assume McMyAdmin is not a Forge mod?)

**Comment by Gunner76th** [comment]
McMyAdmin is a minecraft server management software used to launch, and control various backend functions of the minecraft server, from Ram allocation and whitelisting, to file backups and scheduled tasks. https://www.mcmyadmin.com/

The Prefix issue seems to be beacuse of the fact I am running a pure forge server, and to my knowledge there are no permissions mods out there that are for forge based servers. All of them that I have seen so far are for Bukkit based servers (which as we know, bukkit is dead).  I suspect this is also why the colors are not being changed as well to match what I have set in McMyAdmin. I confirmed this by running the server without EiraIRC installed, and Ops and normal players were white, with no prefix. Restart server with EiraIRC, and Ops are purple regular players are white, both with no prefix.
